packages/commonwealth/test/unit/server_controllers/server_threads_controller.spec.ts

```diff
@@ -345,140 +345,6 @@ describe('ServerThreadsController', () => {
         }),
       ).to.be.rejectedWith('Ban error: big ban err');
     });
-
-    test('should throw error (token balance)', () => {
```

Comment:
If anyone can figure out how we can make this test pass that would be great, otherwise I think it should be removed. Basically the stack trace it provides gives no insight.

Reply:
Yeah I think we might as well remove them if they're creating friction. With the upcoming TRPC refactors, they'll be obsolete anyway.